pkg/reconciler/reconciler.go

```diff
@@ -518,7 +518,7 @@ func WithControllerSetupFunc(f ControllerSetupFunc) Option {
 }
 
 // ControllerSetup allows restricted access to the Controller using the WithControllerSetupFunc option.
-// Currently the only supposed configuration is adding additional watchers do the controller.
+// Currently, the only supposed configuration is adding additional watches to the controller.
```

Comment:
Not sure, but should "supposed" should also be changed to "supported"?
```suggestion
// Currently, the only supported configuration is adding additional watches to the controller.
```